lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -169,12 +173,16 @@ def debug_message
         @events.each do |identifier, event_hashes|
           event_hashes.each { |event| helpful_info << "      [#{identifier}]\t#{event}" }
         end
+      elsif ENV[DISABLE_FETCHING_EVENT_INFO]
+        helpful_info << DISABLED_EVENT_INFO_MESSAGE
```

Comment:
We should probably indent this (and the log message) with "  - " like we do with the actual info. Looks a bit out of place:

![image](https://user-images.githubusercontent.com/4789493/35072000-e28994d2-fbb0-11e7-9245-7e27e36a8659.png)

![image](https://user-images.githubusercontent.com/4789493/35072035-fd5aa22e-fbb0-11e7-8af8-2ecb9fc890ac.png)

(I'm using `test_deploy_result_logging_for_mixed_result_deploy` to see that)